www/index.html

```diff
@@ -465,15 +465,17 @@ <h2 data-i18n="configure-title">Configuration</h2>
                             <button type="button" class="close" data-hide="alert">&times;</button>
                             <span id="persistentMessage"></span>
                         </div>
+                        <!-- Dropdown prev (updated but arrow isn't visible) -->
                         <div class="dropdown" style="float:right">
-                            <select class="form-control" tabindex="4" id="languageSelector" name="languageSelector" aria-labelledby="languageSelector">
-                                <option data-i18n="configure-language-selector-default" selected class="dropdown-item" href="#" value="default">Language</a>
+                            <select class="form-select dropdown-toggle" tabindex="4" id="languageSelector" data-bs-toggle="dropdown" name="languageSelector" aria-labelledby="languageSelector">
+                                <option data-i18n="configure-language-selector-default" selected class="dropdown-item" href="#" value="default">Language </a>
                                 <option class="dropdown-item" href="#" value="en">English</a>
                                 <option class="dropdown-item" href="#" value="fr">Français</a>
                                 <option class="dropdown-item" href="#" value="es">Español</a>
                                 <option data-i18n="configure-language-selector-other" class="dropdown-item" href="#" value="other">More coming soon...!</a>
                             </select>
                         </div>
+                        
```

Comment:
Remove extraneous empty line.

Reply:
Linter on my system causing this on hitting save the format automatically takes it next line.